velodyne_pointcloud/launch/32e_points.launch

```diff
@@ -40,13 +40,11 @@
     <arg name="timestamp_first_packet" value="$(arg timestamp_first_packet)"/>
   </include>
 
-  <!-- start cloud nodelet -->
-  <include file="$(find velodyne_pointcloud)/launch/cloud_nodelet.launch">
+  <!-- start transform nodelet -->
+  <include file="$(find velodyne_pointcloud)/launch/transform_nodelet.launch">
     <arg name="model" value="32E"/>
     <arg name="calibration" value="$(arg calibration)"/>
     <arg name="manager" value="$(arg manager)" />
-    <arg name="fixed_frame" value="$(arg frame_id)" />
-    <arg name="target_frame" value="$(arg frame_id)" />
```

Comment:
Why would you want to remove these instead of exposing them at the top level?

Reply:
My intention was that these parameters are not necessary to get the old CloudNode(let) behavior, because fixed_frame and target_frame were not really used in convert.cc, but just in transform.cc. So the old CloudNode behavior was to publish a non-inertial corrected point cloud in the sensor frame (obtained from the VelodyneScan.msg message). 
In the new implementation this is the default behaviour. So it is not necessary to pass these arguments.
But you're right, maybe we should put them back to show the user, that these options/params do exist. However, this time with empty strings, i.e. `<arg name="target_frame" value="" />`

Reply:
Yes, agreed. Please add them back with empty strings.